backend/src/jwt/decode.js

```diff
@@ -14,6 +14,7 @@ export default async (driver, authorizationHeader) => {
   const session = driver.session()
   const query = `
     MATCH (user:User {id: $id, deleted: false, disabled: false })
+    SET user.lastActiveAt = toString(datetime())
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-02T22:01:40Z" title="Thursday, October 3rd 2019, 12:01:40 am +02:00">Oct 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -14,6 +14,7 @@ export default async (driver, authorizationHeader) => {
   const session = driver.session()
   const query = `
     MATCH (user:User {id: $id, deleted: false, disabled: false })
+    SET user.lastActiveAt = datetime()
```


Actually, can you convert this `datetime()` to a string, at the moment we are doing this cause of a bug with one of the packages we are using for queries 
```suggestion
    SET user.lastActiveAt = toString(datetime())
```